csv2json/csv2json.py

```diff
@@ -0,0 +1,58 @@
+import  json
+import csv
+
+
+def readcsv(fname:str) -> (list):
+    memdata = []
+    with open(fname, newline='', encoding='utf-8') as csvfile:    
+        for item in csvfile:
+            item = item.encode('utf-8').decode('utf-8-sig').strip()
+            data = item.split(',')
+            memdata.append(data)
+            #print(memdata)
+    return memdata
+
+def csv2json(uniquefname:str, commonfname:str) -> (dict):
+    list_uniquedata = []

```

Comment:
The type prefix is redundant. It is probably breaks when the actual type of this variable change in the future. The better way to represent its type is using type hint like following
```suggestion
    unique_data: List[Dict[str, str]] = []
```